jupyterlab_launcher/app.py

```diff
@@ -6,16 +6,14 @@
 from notebook.notebookapp import NotebookApp
 from traitlets import Unicode
 
-from .handlers import add_handlers, LabConfig
+from .handlers import add_handlers
 
 
 class LabLauncherApp(NotebookApp):
 
     default_url = Unicode('/lab',
                           help='The default URL to redirect to from `/`')
 
-    lab_config = LabConfig()
-
```

Comment:
Why remove this? I don't understand where the `LabConfig` is coming from in this case.

Reply:
You're right. I removed it testing stuff and forgot it should go back. Cheers!